.github/workflows/o3_schedule.yaml

```diff
@@ -6,7 +6,21 @@ on:
   schedule:
     - cron:  '0 6 * * MON'
```

Comment:
I'd definitely schedule this multiple times a week! Maybe even twice a day or so :)
But I'm fine to do so later, it's totally up to you.

Reply:
Yes, this part will not stay as it! I think @ggkioulis is already working on this, so for this PR I will leave it ;)

Reply:
Gut 👍